test/os-cluster.test.ts

```diff
@@ -71,6 +71,11 @@ test('Test Resources with security disabled multi-node default instance types',
       Ref: 'managerNodeAsgInstanceProfile1415C2CF',
     },
   });
+  infraTemplate.hasResourceProperties('AWS::AutoScaling::LaunchConfiguration', {
+    MetadataOptions: {
+      HttpTokens: 'required',
```

Comment:
Is httptokens the same as imdsv2?

Reply:
Yes! When you enable IMDSv2, the metadata retrieval needs to be authenticated.